tests/misc_env/install_prereq.py

```diff
@@ -141,18 +144,35 @@ def setup_subscription_manager(ceph, timeout=1800):
     )
     while True:
         try:
-            ceph.exec_command(
-                cmd="sudo subscription-manager --force register  "
-                "--serverurl=subscription.rhsm.stage.redhat.com:443/subscription  "
-                "--baseurl=https://cdn.redhat.com --username=rhcsuser --password=rhcsuser",
-                timeout=720,
-            )
+            command = "sudo subscription-manager --force register --username={username} --password={password} "
```

Comment:
IMO, we could retrieve the contents from the yaml file and then use the appropriate key for extracting the values. For example:
```
config_ = get_cephci_config()
cmd = "subscription-manager --force register"

if is_production:
  username_ = config_["cdn_credentials"]["username"]
  password_ = config_["cdn_credentials"["password"]
else:
  cmd += " --serverurl=subscription.rhsm.stage.redhat.com:443/subscription --baseurl=https://cdn.redhat.com"
  username_ = config_["stage_credentials"]["username"]
  password_ = config_["stage_credentials"]["password"]

cmd += f" --username={username_} --password={password_}"

ceph.exec_command(sudo=True, cmd=cmd, timeout=720)
```

Reply:
Thanks a lot for the suggestions, I've made changes as suggested